docs/_posts/2025-11-04-spec-version-history.md

```diff
@@ -0,0 +1,58 @@
+<!--
+SPDX-FileCopyrightText: 2025 OpenCHAMI Contributors
+
+SPDX-License-Identifier: MIT
+-->
+
+---
+layout: post
+title: "Regenerate to Evolve: Adding Features After Release"
+description: "How updating Fabrica and regenerating code lets you grow existing APIs without rewrites."
+---
+
+APIs live for a long time. They grow as your product grows. The hard part is adding features without breaking what you already shipped. Fabrica is built for this kind of steady change. You keep your resource definitions as the source of truth. The generator keeps the rest in sync.
+
+Here is how it works in practice. Update Fabrica to v0.3.1. That release added opt‑in spec version history. Take a service you built earlier, add the versioning marker to one resource, and run the generator again. The server, storage helpers, client library, and CLI pick up version endpoints and commands. Your custom code stays where it is.
```

Comment:
Do we want to hardcode v0.3.1 or maybe just say like "Update to a version of Fabrica newer than v0.3.1" or something like that? I think for most people (who will be reading this in the future) v0.3.1 will not be an "Upgrade"!